mitodl/micromasters
# Tasawer/course team page

#### Pre-Flight checklist


#### What are the relevant tickets?
https://trello.com/c/kdktSa7B/1-update-course-team-layout

#### What's this PR do?
Update design of course team page.

#### How should this be manually tested?

- Create a program
- Add faculty members
- Add a child page course team tab page
- Render program page, you should see instructors carousel on the program page
- Goto course team tab page, you should not see instructors carousel on the program page

#### Screenshots (if appropriate)
**Desktop**
![screencapture-mm-odl-local-8079-test-tasawer-program-course-team-2020-01-28-16_59_05](https://user-images.githubusercontent.com/4245618/73262362-c8330f00-41ef-11ea-9069-f2ea4969a1ad.png)

**Tablet** 

![screencapture-mm-odl-local-8079-test-tasawer-program-course-team-2020-01-28-16_59_31](https://user-images.githubusercontent.com/4245618/73262363-c8cba580-41ef-11ea-9000-42d4cce6868f.png)

**Mobile**
![screencapture-mm-odl-local-8079-test-tasawer-program-course-team-2020-01-28-17_00_04](https://user-images.githubusercontent.com/4245618/73262364-c8cba580-41ef-11ea-9796-93d2ee059f5b.png)
